app/policies/measure_category_policy.rb

```diff
@@ -2,22 +2,29 @@
 
 class MeasureCategoryPolicy < ApplicationPolicy
   def permitted_attributes
-    [:measure_id,
+    [
       :category_id,
-      measure_attributes: [:title, :description, :target_date, :draft],
-      category_attributes: [:id, :title, :short_title, :description, :url,
-        :taxonomy_id,
+      :measure_id,
+      measure_attributes: [
```

Comment:
@parndt just for my understanding: does this allow creating measures and categories at the same time as a link?